### PR TITLE
Works and aboutme

### PR DESCRIPTION
**Pull request by ritikarawat220**
Portfolio setup - mobile version
-Add indexhtml
-Add css
- Add about myself
- fixed icons
- fixed linters

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -75,6 +92,258 @@ nav {
...
+  text-decoration: none;
+}
+
+.works {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 114px 24px;
+  gap: 88px;
+}
```
Comment (roshan-bajgain) [inline style.css]: - [x] According to the project requirement the main container of the projects should have a `display: grid`. Please do so

File: index.html [hunk 2]
```diff
@@ -54,7 +54,240 @@ <h2 class="contact-title">LET'S CONNECT</h2>
...
+            </li>
+            <!-- /Frameworks -->
+            <li class="my-skill">
+              <div class="skill-container">
+                <h3 class="skill-title">Skills</h3>
+                <button class="btn-chev" type="button">
+                  <img src="./chev-right.svg" alt="" />
+                </button>
+              </div>
+              <div class="separator-line"></div>
```
Comment (roshan-bajgain) [inline index.html:260]: - [x] You should stick to the design as much as possible ie: including color, background, and spacing. Please adjust the spacing for the hx tag also.
### Your design
his

### Figma design
fimga

File: index.html [hunk 3]
```diff
@@ -54,7 +54,240 @@ <h2 class="contact-title">LET'S CONNECT</h2>
...
+              name="message"
+              cols="30"
+              rows="6"
+              placeholder="Write your message here"
+              required
+            ></textarea>
+            <button class="btn-submit" type="submit">Get In Touch</button>
+          </form>
+        </section>
       </main>
```
Comment (roshan-bajgain) [inline index.html]: - [x] [Optional] You can do up to `About myself` section. You are assigned to do this contact form next week. So for now please remove it.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @ritikarawat220 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ritikarawat220** [comment]
> Hi @ritikarawat220
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
> 
> ## Highlights
> * No linter errors. ✔️
> * GitHub flow is followed. ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Its done now.. can you check

**Review by roshan-bajgain — APPROVED** [review]
Hi @ritikarawat220, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**